src/configs/shlvl.rs

```diff
@@ -16,9 +16,9 @@ pub struct ShLvlConfig<'a> {
 impl<'a> RootModuleConfig<'a> for ShLvlConfig<'a> {
     fn new() -> Self {
         ShLvlConfig {
-            threshold: 3,
+            threshold: 2,
             format: "[$symbol$shlvl]($style)",
-            symbol: "↕️ ",
+            symbol: "↕️  ", // extra space for emoji
```

Comment:
Hey @daniel-white, it's time to dive into the past :) **The question: why 2 spaces?** It just looks like too much (asymmetrical):
![image](https://user-images.githubusercontent.com/5967447/202343096-589ae51e-4beb-43d2-87d0-1c23b5917e65.png) 
![image](https://user-images.githubusercontent.com/5967447/202343191-5289650b-da0a-4671-ba53-01589ae9745a.png)

Now, the 1 space (👍GOOD!):
![image](https://user-images.githubusercontent.com/5967447/202343448-eb477f3f-75c4-405b-827f-7cebf370e561.png)
![image](https://user-images.githubusercontent.com/5967447/202343468-8e50d11d-c9c9-48a8-be9f-de0d97cf0de1.png)

Are you okay if I fix it?

Reply:
The macOS emoji